README.md

```diff
@@ -1,12 +1,44 @@
 # bulkfhirloader
```

Comment:
Could you provide some additional information about the parallelization? How many workers are used? Is there a way to configure this?